app/assets/stylesheets/govuk_publishing_components/_all_components.scss

```diff
@@ -1,18 +1,34 @@
+// This is the file that the application needs to include in order to use
+// the components.
+
 // Include all of the GOV.UK Frontend styles. This includes fonts, and individual components.
+
 @import "components/helpers/govuk-frontend-settings";
 
-// This is the file that the application needs to include in order to use
-// the components.
+// Include common imports used by many components
+@import "govuk-frontend/all";
```

Comment:
`/all` will suffice, as it include all the other imports listed below

Reply:
If we remove the redundant `govuk-frontend` imports below I'm happy to get this merged as we're not able to consume the components individually anyway at this point.

Reply:
@alex-ju have done so, can you check? Seems to be alright but I want to be sure (breaking _all_ of the components just before Christmas wouldn't be good).

Reply:
sure, will test it now